gcampus/analysis/templates/gcampusanalysis/components/od_card.html

```diff
@@ -0,0 +1,32 @@
+{% load i18n list_iter %}
+{% with object_list|one_more:forloop.counter0|list_item:row_pos as parameter %}
```

Comment:
Whoah, what does this do? Can you maybe add a small comment (`{# ... #}`) explaining the code?

Reply:
fixed in 16e1c8a4a70527bacea3c35e2da0780da9a1637e